packages/libs/src/sdk/api/challenges/ChallengesApi.ts

```diff
@@ -27,7 +30,7 @@ import {
   GenerateSpecifierSchema
 } from './types'
 
-export class ChallengesApi extends BaseAPI {
+export class ChallengesApi extends GeneratedChallengesApi {
```

Comment:
cc @rickyrombo
This manual implementation wasn't deriving from the generated api (likely because we weren't outputting anything there?). But it has this weird quirk of using an instance of the `full` API below for submitting. Double check me on this file and let me know if we should do it differently.

Reply:
yep this looks good! no need to move the attestations to v1 rn.

Arguably this could extend from the full version but that's not usually what we're doing with other API clients